dask_expr/datasets.py

```diff
@@ -53,42 +70,45 @@ def random_state(self):
                 if self.seed is None
                 else random_state_data(npartitions, self.seed)
             )
-            for k in self.operand("dtypes")
+            for k in self._dtypes
         }
 
     def _filtered_task(self, index):
         full_divisions = self._divisions()
-        column_states = [self.random_state[k][index] for k in self.operand("dtypes")]
+        column_states = [self.random_state[k][index] for k in self._dtypes]
         if self.seed is not None and len(column_states) > 0:
             # These will be the same anyway, so avoid serializing all of them
             column_states = [column_states[0]]
-        return (
+        task = (
             make_timeseries_part,
             full_divisions[index],
             full_divisions[index + 1],
-            self.operand("dtypes"),
-            self.columns,
+            self._dtypes,
+            list(self._dtypes.keys()),
             self.freq,
             column_states,
             self.kwargs,
         )
+        if self._series:
+            return (operator.getitem, task, self.operand("columns")[0])
+        return task
 
     def _simplify_up(self, parent):
-        if isinstance(parent, Projection) and len(self.dtypes) > 1:
-            dtypes = {col: self.operand("dtypes")[col] for col in parent.columns}
-            out = Timeseries(
+        if isinstance(parent, Projection) and is_dataframe_like(self._meta):
+            make_series = is_scalar(parent.operand("columns")) and not self._series
+            return Timeseries(
                 self.start,
                 self.end,
-                dtypes=dtypes,
+                dtypes=self.operand("dtypes"),
```

Comment:
Now that we aren't changing `dtypes` here, then we can remove `Timeseries._simplify_up` altogether. We also don't need the `_token_dtypes` operand anymore.

Reply:
Yeah that's fair.

We still need the token. We have to be able to distinguish between different time series calls

```
df = timeseries()
df2 = timeseries()

(df + df2) != 2*df
```

Reply:
>We have to be able to distinguish between different time series calls

Wont the seeds be different (making them distinguishable anyway)?

Reply:
I have another PR to remove the seed, specifying a seed makes serialising the graph very slow (should probably optimise this, but didn't have time yet) and is inconsistent with dask/dask, which is what makes benchmarking hard

Reply:
We should be good here though, the randint call is only triggered when constructing the graph, so not when we go through optimize